src/python/test/test_cover_complex.py

```diff
@@ -85,3 +87,42 @@ def test_voronoi_graph_induced_complex():
     assert stree.num_vertices() == 2
     assert (stree.num_simplices() - stree.num_vertices()) == 1
     assert stree.dimension() == 1
+
+def test_cover_complex():
+
+    #              x
+    #              |
+    #              x
+    #              |
+    #      x---x---x
+    #      |       |
+    #      x       x
+    #      |       |
+    #      x---x---x
+    #      |
+    #      x
+    #      |
+    #      x
+
+    X = np.array([[1,1],[1,1.5],[1,2],[1,2.5],[1,3],[1.5,2],[1.5,3],[2,2],[2,2.5],[2,3],[2,3.5],[2,4]])
+    F = np.array([[1,1,1,1,1,1.5,1.5,2,2,2,2,2],[1,1.5,2,2.5,3,2,3,2,2.5,3,3.5,4]]).T
+
+    M = CoverComplex(complex_type="gic", input_type="point cloud", cover="functional", colors=None, mask=0, filters=F[:,1], filter_bnds=np.array([.5,4.5]), 
+         resolutions=np.array([4]), gains=np.array([.3]), graph='rips', rips_threshold=.6).fit(X)
+
+    assert list(M.simplex_tree.get_filtration()) == [([0], 0.0), ([1], 0.0), ([0, 1], 0.0), ([2], 0.0), ([1, 2], 0.0), ([3], 0.0), ([2, 3], 0.0)]
+
+    M = CoverComplex(complex_type="gic", input_type="point cloud", cover="voronoi", voronoi_samples=5, colors=None, mask=0, filters=F[:,1], filter_bnds=np.array([.5,4.5]), 
+         resolutions=np.array([4]), gains=np.array([.3]), graph='rips', rips_threshold=.6).fit(X)
+
+    assert list(M.simplex_tree.get_filtration()) == [([0], 0.0), ([1], 0.0), ([0, 1], 0.0), ([2], 0.0), ([1, 2], 0.0), ([3], 0.0), ([1, 3], 0.0), ([4], 0.0), ([2, 4], 0.0), ([3, 4], 0.0)]
```

Comment:
This test fails on my computer:
```txt
>       assert list(M.simplex_tree.get_filtration()) == [([0], 0.0), ([1], 0.0), ([0, 1], 0.0), ([2], 0.0), ([1, 2], 0.0), ([3], 0.0), ([1, 3], 0.0), ([4], 0.0), ([2, 4], 0.0), ([3, 4], 0.0)]
E       assert [([0], 0.0), ...3], 0.0), ...] == [([0], 0.0), ...3], 0.0), ...]
E         At index 6 diff: ([0, 3], 0.0) != ([1, 3], 0.0)
```

Reply:
Mmmh that's weird, it doesn't on my laptop. A reason might be that you were super unlucky and the 5 voronoi samples were picked so poorly that you missed the structure. Does it still fail?